docs/book/emails.rst

```diff
@@ -7,73 +7,120 @@ E-Mails
 Sylius is sending various e-mails and this chapter is a reference about all of them. Continue reading to learn what e-mails are sent, when and how to customize the templates.
 To understand how e-mail sending works internally, please refer to :doc:`SyliusMailerBundle documentation </bundles/SyliusMailerBundle/index>`.
 
-User Confirmation E-Mail
-------------------------
+User Confirmation
+-----------------
 
-Every time new customer registers via registration form or checkout, ``user_confirmation`` e-mail is sent to him.
-The default template is
+Every time a new customer registers via the registration form, a user confirmation e-mail is sent to them.
```

Comment:
Every time a customer registers via the registration form, a user confirmation e-mail is sent to them.